### ISSUE TITLE
Handling of new line characters

### ISSUE THREAD

**Issue by punitganshani** [EvotecIT/OfficeIMO#37]
Right now if the text contains `NewLine` character, the rendering as `Text` or `Paragraph` ignores the newline and renders its as plain text. 

If we manually add `Paragraph` there is additional BREAK (ENTER) rendered which is not rendered if we press SHIFT+ENTER in Word. 

Example

```
                    var json = JValue.Parse(oldJson).ToString(Formatting.Indented);
                    string[ ] newLineArray = { Environment.NewLine };
                    string[ ] textArray = json.Split( newLineArray, StringSplitOptions.None );
                    for (int index = 0; index < textArray.Length; index++)
                    {                             
                        document.AddParagraph(textArray[index]);          
                    }
```

This will render the Word Document as below,

![image](https://user-images.githubusercontent.com/1032553/193556187-f89917b3-d9ea-4ec5-8a37-89a82f75a1fe.png)

What is expected is

![image](https://user-images.githubusercontent.com/1032553/193556289-65fba165-babf-4e75-9ae2-cb12d0fdcd3c.png)

Please suggest what is the best way to replace an `ENTER` with a `SHIFT+ENTER`

**Comment by PrzemyslawKlys** [comment]
So - the difference between ENTER and SHIFT+ENTER on the XML tells us what the issue is about:

- Enter
![image](https://user-images.githubusercontent.com/15063294/194371751-a136126e-fd52-4f2b-9a6e-5ed38904bc6d.png)

- SHIFT+ENTER
![image](https://user-images.githubusercontent.com/15063294/194371804-a573630f-38c2-44bc-93ef-ffd8a058dc1e.png)

I would probably need to add new method for paragraph AddBreak to allow you to use it. Unless you would be willing to do that

**Comment by PrzemyslawKlys** [comment]
Currntly break is exposed but for Document, and not Paragraph. 

https://github.com/EvotecIT/OfficeIMO/blob/7010fd22e141db5880094a6d61ee953cd3b2c2e3/OfficeIMO.Word/WordDocument.PublicMethods.cs#L23-L44

**Comment by PrzemyslawKlys** [comment]
![image](https://user-images.githubusercontent.com/15063294/194375619-a43d0d20-50b9-445a-a8c7-d778a4c3fac2.png)

```csharp
public static void Example_BasicWordWithBreaks(string folderPath, bool openWord) {
    Console.WriteLine("[*] Creating standard document with paragraph & breaks");
    string filePath = System.IO.Path.Combine(folderPath, "BasicDocumentWithParagraphsAndBreaks.docx");
    using (WordDocument document = WordDocument.Create(filePath)) {
        var paragraph1 = document.AddParagraph("Adding paragraph1 with some text and pressing ENTER");


        var paragraph2 = document.AddParagraph("Adding paragraph2 with some text and pressing SHIFT+ENTER");
        paragraph2.AddBreak();
        paragraph2.AddText("Continue1");
        paragraph2.AddBreak();
        paragraph2.AddText("Continue2");

        var paragraph3 = document.AddParagraph("Adding paragraph3 with some text and pressing ENTER");


        document.Save(openWord);
    }
}
```

Would that work? 

![image](https://user-images.githubusercontent.com/15063294/194375849-7009d02a-f107-4a87-be5b-9f11d71c4aa1.png)

```csharp
public static void Example_BasicWordWithBreaks(string folderPath, bool openWord) {
    Console.WriteLine("[*] Creating standard document with paragraph & breaks");
    string filePath = System.IO.Path.Combine(folderPath, "BasicDocumentWithParagraphsAndBreaks.docx");
    using (WordDocument document = WordDocument.Create(filePath)) {
        var paragraph1 = document.AddParagraph("Adding paragraph1 with some text and pressing ENTER");


        var paragraph2 = document.AddParagraph("Adding paragraph2 with some text and pressing SHIFT+ENTER");
        paragraph2.AddBreak();
        paragraph2.AddText("Continue1");
        paragraph2.AddBreak();
        paragraph2.AddText("Continue2");

        var paragraph3 = document.AddParagraph("Adding paragraph3 with some text and pressing ENTER");

        var paragraph4 = document.AddParagraph("Adding paragraph4 with some text and pressing SHIFT+ENTER");
        paragraph4.AddBreak();

        document.Save(openWord);
    }
}
```

Which looks like this

![image](https://user-images.githubusercontent.com/15063294/194376217-ce9a4ab3-eb32-450c-b76e-bb12b277df21.png)

**Comment by PrzemyslawKlys** [comment]
Will be available in 0.3.0

**Comment by PrzemyslawKlys** [comment]
Added a way to use NewLines directly in text: 
- https://github.com/EvotecIT/OfficeIMO/pull/127
